.github/workflows/deploy-to-pypi.yml

```diff
@@ -32,7 +52,8 @@ jobs:
     #    password: ${{ secrets.TEST_PYPI_API_TOKEN }}
     #    repository_url: https://test.pypi.org/legacy/
     - name: Publish distribution 📦 to PyPI
-      if: startsWith(github.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@master
+      if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main')
```

Comment:
isn't it going to be pushed on pypi for each commit?

Reply:
Hmmm you're right. Bad copy/paste...

Reply:
Fixed. Thanks!